website/docs/adopting.md

```diff
@@ -161,6 +161,11 @@ enable. The tl;dr is that type checking happens when we add
 [`# typed: true`](static.md) to files and write [method signatures](sigs.md) in
 those files.
 
+## Step 6: Source Control
+
+All files generated by `srb init` should be committed to source control, 
+including the `Makefile` and the entire `sorbet` directory.
```

Comment:
there shouldn't be a `Makefile`, only the `sorbet` directory. Maybe one of your files made that?